gsub.js

```diff
@@ -38,7 +38,7 @@ const initGsubTables = (_dom, options) => {
   }
 };
 
-const buildGsubTables = (_dom, ligature, options) => {
```

Comment:
ESLint was marking this as an unused variable.

Reply:
Interesting. That is exported on line 312.

Reply:
Sorry for not being clear.  I was referring to just the `options` argument.  The `buildGsubTables` function is still used and that I left alone.

Reply:
Ah, you're right. `options` was used in a previous commit 6f3d00ae6d5926506b102b70a667088df422d4e7, but removed in 321486d19ea3fe1916e7e13f68853f0da51d2c4

Apparently I forgot to remove the parameter.